VideoSyncClass/VideoSync.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from subprocess import run, call
+from pathlib import Path
+import yaml
+import os
+from scipy.io import wavfile
+from scipy.io.wavfile import read as wavread
+import mne
+
+class VideoSync:
+    def __init__(self, input_folder=None, output_folder="/output"):
+        """
+        Summary: VideoSync represents an object performing synchronization between multiple streams of data.
```

Comment:
there are standards around line length and the content and phrasing of docstrings that are worth knowing about. `pydocstyle` is the thing to search here, though again I think maybe `ruff` does these too?